Lesson1 (Sequence containers)/Gerus Roman/Homework_2/Task_1/Brackets_valid.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+#include <stack>
+
+bool bracketsCheck (std::stack<char> & stack_with_brackets) {
```

Comment:
**you didn't get the task point.
what the profit of stack here?**
it is absolutely useless.